code/src/Components/ThoughtForm.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import 'tachyons'
+
+const ThoughtForm = ({newMessages, onSetThoughtChange, onFormSubmit}) => {
+  return (
+    
+    <form className="grow shadow-20" onSubmit={onFormSubmit}>
+      <h3>Any good thoughts to share today ?</h3>
+
+      <textarea 
+      placeholder='Any thoughts...' 
+      rows="3" cols="30" 
+      value={newMessages} 
+      onChange={onSetThoughtChange}
+      />
```

Comment:
when testing for accessibility the Wave report suggests using a label element or an aria description for the textarea element